subprojects/docs/src/docs/release/notes.md

```diff
@@ -109,6 +109,14 @@ repositories {
 ```
 See [the documentation](userguide/kotlin_dsl.html#type-safe-accessors) for details.
 
+### Dependency verification improvements
+
+[Dependency verification](userguide/dependency_verification.html) is a feature that allows build to verify the checksums and signatures of the plugins and dependencies that are used by your project.
+
+With this release, the generation of the dependency verification file has been improved to produce stable output.
+This allows you to leverage generation as an update strategy for the verification metadata when dependencies change in your project.
```

Comment:
This is not entirely clear. Can we add some context on the "generation update strategy"?